scripts/daily_snapshot/.env

```diff
@@ -1,8 +1,8 @@
 # Definitely need to define these:
-AWS_ACCESS_KEY_ID=
-AWS_SECRET_ACCESS_KEY=
-SLACK_API_TOKEN=
-SLACK_NOTIF_CHANNEL=
+AWS_ACCESS_KEY_ID=D4RGUYZ5CCJK7NQGR5CB
+AWS_SECRET_ACCESS_KEY=zxc0YAh+dMmklQbFWvf1+K303iza+MXcb4O/Lnc1+5A
+SLACK_API_TOKEN=xoxb-160325419412-3252853891664-0o8JmYrHCQKRRlxO9hHwPGCv
```

Comment:
You have to be very careful about not committing private keys. Now we have to invalidate those keys.

Reply:
I am surprised at this because I actually didnt create any new keys, I am not working with AWS, I am working with DO so i am not sure I know how this came up in my processes